content/07.pathogenesis.md

```diff
@@ -223,6 +223,7 @@ One case study [@doi:10/gg4sd6] described an adult who appeared to show symptoms
 The presentation of SARS-CoV-2 infection is therefore likely to be largely distinct between adult and pediatric populations.
 However, not all cases of severe COVID-19 in children are characterizable as MIS-C.
 A recent study [@doi:10.1001/jama.2021.2091] described demographic and clinical variables associated with MIS-C in comparison with non-MIS-C severe acute COVID-19 in young people in the United States.
+Some cases of COVID-19 infection in children have involved various neurological complications, which were usually transient but on rare occasions were life-threatening or fatal [@doi:10.1001/jamaneurol.2021.0504]. 
```

Comment:
This is a good sentence and study to add.  However, I'm not sure about the placement.  Putting it after the sentence
> not all cases of severe COVID-19 in children are characterizable as MIS-C

may suggest these cases do not involve MIS-C.  However, almost half the patients with "life-threatening neurologic involvement..." "met criteria for MIS-C".

Reply:
I didn't realize that but you are right!  I will try moving it to earlier in the paragraph.